csp.ipynb

```diff
@@ -1205,15 +1195,13 @@
    "source": [
     "### Helper Functions\n",
     "\n",
-    "We will now implement a few helper functions that will help us visualize the Coloring Problem. We will make some modifications to the existing Classes and Functions for additional book keeping. To begin we modify the **assign** and **unassign** methods in the **CSP** to add a copy of the assignment to the **assignment_history**. We call this new class **InstruCSP**. This will allow us to see how the assignment evolves over time."
+    "Now we will implement a few helper functions that will allow us to visualize the Coloring Problem; we'll also make a few modifications to the exisiting classes and functions for additional record keeping. To start with, we modify the **assign** and **unassign** methods in the **CSP** in order to add a copy of the assignment to the **assignment_history**. We name this new class as **InstruCSP**; it will allow us to see how the assignment evolves over the time. "
```

Comment:
- 'We will now...' was perfectly alright
- You've misspelled 'existing'
- 'To begin with' sounds a lot better than 'To start with'
- There can't be an 'as' after 'We name this new class...'
- Time cannot have an article in this context

Reply:
Agreed. Thanks for pointing these out.